src/rgw/rgw_rest_role.cc

```diff
@@ -13,15 +13,68 @@
 
 #include "rgw_common.h"
 #include "rgw_op.h"
+#include "rgw_process_env.h"
 #include "rgw_rest.h"
-#include "rgw_role.h"
+#include "rgw_rest_conn.h"
 #include "rgw_rest_role.h"
+#include "rgw_role.h"
 #include "rgw_sal.h"
 
 #define dout_subsys ceph_subsys_rgw
 
 using namespace std;
 
+int forward_iam_request_to_master(const DoutPrefixProvider* dpp,
```

Comment:
@cbodley : Just a question here - if in future we plan to add multisite replication for `oidc-provider` in rgw, where should this method be shifted to? `rgw_rest_iam.h`?

Reply:
rgw_rest_conn.* might be a good place since these functions depend on RGWRESTConn